### PR TITLE
resolves #4426

### LINKED ISSUE #4426: Avoid malware
Why no mention of not using old binary / 2003 file formats, use the new 2007 X-file formats.
No mention of not using RTF format

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3c1cf16e-b893-5928-bab5-18dab0435316
* Version Independent ID: 25af1ac9-b386-1b11-4954-b246e4a20ef1
* Content: [Macro malware](https://docs.microsoft.com/en-us/windows/security/threat-protection/intelligence/macro-malware#feedback)
* Content Source: [windows/security/threat-protection/intelligence/macro-malware.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/intelligence/macro-malware.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @levinec
* Microsoft Alias: **levinec**

### PR DESCRIPTION
**Pull request by martyav**
Issue #4426, Avoid malware

> The Macro Malware page is very brief, I was thinking some more information would help. At the end you have <snip> For more general tips, see prevent malware infection<https://docs.microsoft.com/en-us/windows/security/threat-protection/intelligence/prevent-malware-infection>. </snip> Since much macro malware is delivered as part of Phishing attempts, I was thinking if you added a link to the phishing page would help shorten the users search for specific information ie <snip> For more specific tips see https://docs.microsoft.com/en-us/windows/security/threat-protection/intelligence/phishing </snip>

I added the link to the phishing page. Although the general tips page also include a link to the phishing page, a shorter path to what the reader needs couldn't hurt?

### CODE REVIEW

**Comment by jvsam** [comment]
Hi @martyav thanks for pushing this PR. Can you merge the updates or does this have to go through the normal review and approval process? Thank you.

**Comment by martyav** [comment]
I can merge it myself, just looking to hear if there were any serious objections or suggestions on wording from other writers on the repo.

**Comment by jvsam** [comment]
I see, thanks for the clarification @martyav. 

@nenonix and @JohanFreelancer9 for copy review please. Thank you.

File: windows/security/threat-protection/intelligence/macro-malware.md [hunk 1]
```diff
@@ -45,4 +45,6 @@ We've seen macro malware download threats from the following families:
 
 * Enterprises can prevent macro malware from running executable content using [ASR rules](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction#enable-and-audit-attack-surface-reduction-rules)
 
-For more general tips, see [prevent malware infection](prevent-malware-infection.md).
+For more tips on protecting yourself from supicious emails, see [phishing](phishing.md).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/intelligence/macro-malware.md]:
```suggestion
For more tips on protecting yourself from suspicious emails, see [phishing](phishing.md). 
```

**Comment by martyav** [comment]
There don't seem to be any objections, and since it's not a huge change, I'm merging

**Comment by jvsam** [comment]
Thanks for merging @martyav!

**Pull request merged**